documentation/rules-namingconvention.adoc

```diff
@@ -0,0 +1,13 @@
+=== Naming Convention Rules
```

Comment:
All headers for rule documents should be on level two, so starting with "==", except for component, layer and scope rules, to be consistent with the changes from your last commits.